dev/com.ibm.ws.transport.http/src/com/ibm/ws/http/channel/internal/HttpBaseMessageImpl.java

```diff
@@ -2905,6 +2906,14 @@ else if (getServiceContext().getHttpConfig().onlySameSiteStar()) {
                 }
             }
 
+            if (cookie.getAttribute("samesite") != null && cookie.getAttribute("samesite").equals(HttpConfigConstants.SameSite.NONE.getName())) {
+                String userAgent = getServiceContext().getRequest().getHeader(HttpHeaderKeys.HDR_USER_AGENT).asString();
+                if (SameSiteCookieUtils.isSameSiteNoneIncompatible(userAgent)) {
+                    //TODO: do we remove Secure, probably should be retained.
```

Comment:
At this point in time there is probably no way to know if Session/WC/Security etc added the Secure attribute or if it was already set correct?